tests/test_fakes.py

```diff
@@ -938,12 +947,20 @@ def test_show_tables(dcur: snowflake.connector.cursor.SnowflakeCursor):
             "kind": "TABLE",
             "database_name": "DB1",
             "schema_name": "SCHEMA1",
+            "comment": None,
         },
     ]
     # assert dcur.fetchall() == objects
     dcur.execute("show terse tables in db1.schema1")
     assert dcur.fetchall() == objects
-    assert [r.name for r in dcur.description] == ["created_on", "name", "kind", "database_name", "schema_name"]
+    assert [r.name.lower() for r in dcur.description] == [
+        "created_on",
+        "name",
+        "kind",
+        "database_name",
+        "schema_name",
+        "comment",
```

Comment:
likewise here, `terse tables` doesn't have the comment column
```suggestion
    assert [r.name for r in dcur.description] == ["created_on", "name", "kind", "database_name", "schema_name"]
```